docs/0.3/using_purchase_order.md

```diff
@@ -230,102 +214,72 @@ public key file in the next command.
     gnrl $(cat ~/.grid/keys/beta-agent.pub) \
     --active \
     --role admin \
-    --role po-manager`
+    --role po-manager
     ```
 
 ### Create Purchase Orders
 
 {:start=1}
 
 1.  Using the `gridd-beta` command line, propose a draft purchase order on behalf
-    of Fred's Farm.
+    of MyOrg. An example purchase order XML file can be downloaded with this link:
+    <a href="/docs/0.3/purchase_order/test_po.xml" download="purchase_order.xml">
+Example PO XML</a>.
 
     ```
     root@gridd-beta:/# grid po create \
-    --org farm \
-    --id po_number: 0123456 \
-    --workflow-status Issued \
+    --buyer-org gnrl \
+    --seller-org myorg \
+    --alternate-id po_number:0123456 \
+    --workflow-state create \
+    --workflow-id built-in::collaborative::v1 \
     --wait 100
 
     root@gridd-beta:/# grid po version create \
-    --org farm \
-    --po 0123456 \
+    po_number:0123456 \
+    1 \
     --order-xml <path to order> \
     --draft \
-    --workflow-status Editable \
-    --wait 100 \
-    v1
-    ```
-
-1.  Using the `gridd-alpha` command line, accept the proposed purchase order
-
-    ```
-    root@gridd-alpha:/# grid po version update \
-    --org farm \
-    --po po_number:0123456 \
-    --not-draft \
-    --workflow-status Proposed \
-    --wait 100 \
-    v1
-
-    root@gridd-alpha:/# grid po update \
-    --org farm \
-    --workflow-status Issued \
-    --accepted-version v1 \
+    --workflow-state create \
```

Comment:
this will actually need to be "issued", we're getting rid of using the 'create' state like this

Reply:
same with the version created below, rather than create it will be "proposed" (or "editable" if we're making a draft version)